### PR TITLE
Contact Speaker

### LINKED ISSUE #23: Contact speaker process
After clicking the email icon on the speaker card, what next? 

We'll need a form to collect:

- Sender Name
- Sender Email
- Sender Message

And after submission, where do we store it? Another AirTable?

### PR DESCRIPTION
**Pull request by ann-kilzer**
Resolves #23 

Collect name, email, and a short message from the user. 

web

web

TODO in future PRs: 

- [x] Form Validation
- [x] Wire to Netlify. [This post](https://www.netlify.com/blog/2018/09/07/how-to-integrate-netlify-forms-in-a-vue-app/) may help, though simply sending a POST using axios might be straightforward as well.

### CODE REVIEW

File: web/src/components/CardHeader.vue [hunk 1]
```diff
@@ -3,20 +3,21 @@
     <h2>{{ title }}</h2>
     <v-spacer />
     <v-btn
+      v-if="false"
```
Comment (ann-kilzer) [inline web/src/components/CardHeader.vue:6]: I caught this mistake though it's a bit outside the PR. The button was rendering even though the icon is disabled.

**Comment by tuttiq** [comment]
By the way Wiring to Netlify is pretty straightforward according to [their docs](https://docs.netlify.com/forms/setup/#html-forms)

**Comment by ann-kilzer** [comment]
Yeah I already saw their posts and linked one in the ticket description. Sure, it might be straightforward with vanilla Vue, but I am not sure how it will play with Vuetify. I need to look further.

File: web/vue.config.js [hunk 2]
```diff
@@ -1,5 +1,25 @@
+const path = require('path');
+const PrerenderSPAPlugin = require('prerender-spa-plugin');
+
 module.exports = {
   transpileDependencies: [
     'vuetify',
   ],
+  configureWebpack: () => {
```
Comment (ann-kilzer) [inline web/vue.config.js:8]: We have to pre-render certain pages for Netlify to read the form.

Comment (ann-kilzer) [reply web/vue.config.js]: This didn't work. Adding a hidden form in the static html did, however!

**Comment by ann-kilzer** [comment]
Something is off with the speaker.name after the merge 🤔

**Pull request merged**